src/store/newGroupConversationStore.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * @copyright Copyright (c) 2019 Marco Ambrosini <marcoambrosini@pm.me>
+ *
+ * @author Marco Ambrosini <marcoambrosini@pm.me>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+import isEqual from 'lodash/isEqual'
+
+const state = {
+	selectedParticipants: [],
+}
+
+const getDefaultState = () => {
+	return {
+		selectedParticipants: [],
+	}
+}
+
+const getters = {
+	/**
+	 * Gets the selected participants array
+	 * @param {object} state the state object.
+	 * @returns {array} the selected participants array
```

Comment:
Want to explain how such an object looks like (since you also got it wrong and thought it returns `userId`